src/spikeinterface/widgets/amplitudes.py

```diff
@@ -31,7 +41,7 @@ class AmplitudesWidget(BaseWidget):
         (matplotlib backend)
     bins : int or None, default: None
         If plot_histogram is True, the number of bins for the amplitude histogram.
-        If None this is automatically adjusted
+        If None, use 100 bins.
```

Comment:
```suggestion
        If None, uses 100 bins.
```
right? the function is the singular subject?  or you're commanding it I guess.